admin_guide/sdn_troubleshooting.adoc

```diff
@@ -0,0 +1,547 @@
+Fixing that is outside the scope of this document.
+
+You should see one or more router endpoints listed, but that won't
+tell you if they are running on the machine with the given external IP
+address, since the endpoint IP address will be one of the pod
+addresses that is internal to the cluster.  To get the list of router
+host IP addresses, run:
+....
+# oc get pods --all-namespaces --selector=router --template='{{range .items}}HostIP: {{.status.hostIP}}   PodIP: {{.status.podIP}}{{end}}{{"\n"}}'
+HostIP: 192.168.122.202   PodIP: 10.1.0.4
+....
+
+You should see the host IP that corresponds to your external address.
+If you do not, please refer to the
+link:architecture/core_concepts/routes.html[router documentation] to
+configure the router pod to run on the right node (by setting the
+affinity correctly) or update your DNS to match the IP addresses where
+the routers are running.
+
+At this point in the guide, you should be on a node, running your
+router pod, but you still can not get the http request to work.  First
+we need to make sure that the router is mapping the external URL to
+the correct service, and if that works, we need to dig into that
+service to make sure that all endpoints are reachable.
+
+Let's list all of the routes that OpenShift knows about:
+....
+# oc get route
+NAME              HOST/PORT         PATH      SERVICE        LABELS    TLS TERMINATION
+route-unsecured   www.example.com   /test     service-name
+....
+
+If the hostname and path from your URL don't match anything in the
+list of returned routes, then you need to add a route.  See the link:architecture/core_concepts/routes.html[router documentation].
+
+If your route is present, then you need to debug access to the
+endpoints.  That's the same as if you were debugging problems with a
+service, so please continue on with the next <<debugging-a-service>>
+section.
+
+
+== Debugging a Service
+
+If you can't communicate with a service from inside the cluster
+(either because your services can't communicate directly, or because
+you are using the router and everything works until you get into the
+cluster) then you need to work out what endpoints are associated with
+a service and debug them.
+
+First, let's get the services:
+....
+# oc get services --all-namespaces
+NAMESPACE   NAME              LABELS                                    SELECTOR                  IP(S)            PORT(S)
+default     docker-registry   docker-registry=default                   docker-registry=default   172.30.243.225   5000/TCP
+default     kubernetes        component=apiserver,provider=kubernetes   <none>                    172.30.0.1       443/TCP
+default     router            router=router                             router=router             172.30.213.8     80/TCP
+....
+
+You should see your service in the list.  If not, then you need to
+define your
+link:architecture/core_concepts/pods_and_services.html[service].
```

Comment:
Should be:

`link:../architecture/core_concepts/pods_and_services.html[service]`